### PR TITLE
Fix various typo

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.ts,./libraries,*.dxf,*.lff,*.desktop -L ans,atleast,ba,bload,ded,fo,inout,lightyear,recources,te,trun`

### CODE REVIEW

File: lckernel/cad/storage/entitycontainer.h [hunk 1]
```diff
@@ -281,7 +281,7 @@ class EntityContainer {
     /**
      * @brief entitiesWithinAndCrossingAreaFast
      * Find all entities within a selected area.
-     * Unlike entitiesWithinAndCrossingArea it this will return entities based on it's boundig box.
+     * Unlike entitiesWithinAndCrossingArea it this will return entities based on it's bounding box.
```
Comment (UnityOfFairfax) [inline lckernel/cad/storage/entitycontainer.h:284]: “it's” should be “its”

Comment (luzpaz) [reply lckernel/cad/storage/entitycontainer.h]: Fixed in ee8f43b

File: lcviewernoqt/drawitems/lcvtext.cpp [hunk 2]
```diff
@@ -32,8 +32,8 @@ void LCVText::draw(LcPainter& painter, const LcDrawOptions &options, const lc::g
     double alignY = 0.0;
 
     //    double alignX, alignY;
-    // The idea of height() * .2 is just a average baseline offset. Don't this value to seriously,
-    // we could get it from font exists but that sounds over exaggerating for the moment.
+    // The idea of height() * .2 is just an average baseline offset. Don't take this value to seriously,
```
Comment (flywire) [inline lcviewernoqt/drawitems/lcvtext.cpp:35]: **too** seriously

Comment (luzpaz) [reply lcviewernoqt/drawitems/lcvtext.cpp]: Fixed by lordofbikes in 6b27f47

**Comment by luzpaz** [comment]
Found a few follow-up typos in dbf1cf

**Review by lordofbikes — APPROVED** [review]

**Comment by lordofbikes** [comment]
many thanks :+1:

File: lckernel/cad/interface/splitable.h [hunk 3]
```diff
@@ -6,7 +6,7 @@
 namespace lc {
 namespace entity {
 /**
-  * Any entity that we can be splitted into multiple entities
+  * Any entity that we can be split into multiple entities
   *
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply lckernel/cad/interface/splitable.h:10]: `Any entity which can be split into multiple entities` sounds best

**Pull request merged**